test/dropbox/run.sh

```diff
@@ -6,7 +6,11 @@ cd "$(dirname "$0")"
 . ../util.sh
 current_test=$(basename $(pwd))
 
-docker compose up -d
+cp user_v2.yaml user_v2_ready.yaml
```

Comment:
I'd still either:
- gitignore this file
- copy over to a temp file (you can `mktemp` to get one) and mount the tempfile instead

Else, someone will check it in at some point.

Reply:
Oh, completely forgot that. Working on two branches simultaneously and multiple things didn't work well this time. Will add it.